Functions/Logarithm.py

```diff
@@ -0,0 +1,22 @@
+from Power_Function import power_function
+
+#Goal: Calculate the natural logarithm, log base e, of a given input x
+#Method: we use a series based on the area hyperbolic tangent function
+def ln(args):
+
+    x = 0
+
+    # If the right number of arguments are passed, then continue
+    if(len(args) == 1):
+        x = args[0]
+    else:
+        raise Exception(f"Invalid number of arguments, ln got {len(args)} but expected 1.")
```

Comment:
Syntax error here, remove the f?

Reply:
You mean before the string? the f enables string interpolation, so whenever you see curly braces within the string it'll replace that with the value of the variable.

The function was working for me within the test script I wrote, did you try running that? Can you give an example of how you're calling ln?

Reply:
ahh I just checked, it's not working since this branch does not have a working decimal power function implementation. I'll rebase on master and force push the modified history

Reply:
Ok it's done, you should be able to test the function from the CLI now.

Reply:
Ah, you're right. I was actually running the code on an older version of python which could compile f strings. Thanks for the clarification. LGTM! Go ahead and merge it!